java/src/test/java/ai/rapids/cudf/ColumnVectorTest.java

```diff
@@ -6693,6 +6693,9 @@ void testApplyBooleanMaskFromListOfStructure() {
     }
   }
 
+  /**
+   * The caller needs to make sure to close the returned ColumnView
```

Comment:
Local variables in this function are still not closed.

Reply:
Humnn, if this function is always called in a `try` block then we won't this that.